tests/utils_tests/test_module_loading.py

```diff
@@ -188,28 +188,17 @@ class TestFinder:
     def __init__(self, *args, **kwargs):
         self.importer = zipimporter(*args, **kwargs)
 
-    def find_module(self, path):
-        importer = self.importer.find_module(path)
-        if importer is None:
+    def find_spec(self, path, target=None):
+        spec = self.importer.find_spec(path, target)
+        if spec is None:
             return
-        return TestLoader(importer)
-
-
-class TestLoader:
-    def __init__(self, importer):
-        self.importer = importer
-
-    def load_module(self, name):
-        mod = self.importer.load_module(name)
-        mod.__loader__ = self
-        return mod
+        return spec
```

Comment:
This following is just the same as `return spec`:
```python
        if spec is None:
            return
        return spec
```
So:
```python
    def find_spec(self, path, target=None):
        return self.importer.find_spec(path, target)
```

Reply:
Updated.